### ISSUE TITLE
📊 Add SEO image for homepage

### ISSUE THREAD

**Issue by joseph-allen** [nunocoracao/blowfish#1491]
All posts generate with SEO thumbnails, but linking to the thumbnail seems to pull the logo by default, which doesn't look normal for my website.

One line can be added to partials/head.html changing this.
Combining with one of teh config params.

**Comment by nunocoracao** [comment]
Hello @joseph-allen can you give me an example of what you would be expecting to see in the final code?

**Comment by joseph-allen** [comment]
I've put
`<meta property="og:image" content="/posts/aquariums-and-japan/fuji.jpg">`
Just to use an asset served in a blog post.

Something like
`    <meta property="og:image" content={{ .Site.Language.Params.seoImage }}>` in layouts/partials/head.html

And adding that property in params.toml with a default.

The blowfish site has no image https://www.opengraph.xyz/url/https%3A%2F%2Fblowfish.page%2F in these cases some social media will lift any image on the page, which for me is my logo.

This is on Facebook messenger
![Screenshot 2024-05-27 at 11 32 27 am](https://github.com/nunocoracao/blowfish/assets/7880824/b78beccc-1a45-4d62-b9ed-00f6bc16af04)

**Comment by nunocoracao** [comment]
@joseph-allen so this is just for the image tag for the homepage specifically?
sorry just want to double check

**Comment by joseph-allen** [comment]
Yes, I think all other pages, or at least the posts generate a SEO image automatically.

**Comment by rothgar** [comment]
I was looking for this feature too. I realized you you put a `featured.png` in the content folder hugo automatically use it for the open graph image on the home page.
